js/interface.js

```diff
@@ -699,6 +699,10 @@ function submissionBuild(appSubmission, origin) {
     }
 
     Fliplet.Studio.emit('refresh-app-submissions');
+    Fliplet.Studio.emit('app-launch', {
```

Comment:
wouldn't be better to track a generic `track-event` type so you can track any analytics event from widgets?

Reply:
@squallstar There already is one. In this case, I had to run assessments, so I opted out for using that method and chose to go through an `updateAppLaunch` function that determines what events need to be fired.